hoomd/md/ActiveForceComputeGPU.cu

```diff
@@ -57,57 +56,21 @@ __global__ void gpu_compute_active_force_set_forces_kernel(const unsigned int gr
 
     unsigned int tag = d_groupTags[group_idx];
     unsigned int idx = d_rtag[tag];
```

Comment:
Remove `m_groupTags` and access the indices directly from the group via `m_group->getIndexArray()`.
Now that the directions are tied to orientation, the computation does not need the particle tag.